doc/General/Contributing.md

```diff
@@ -131,9 +136,7 @@ project.
       ```ssh
       git subtree add --squash --prefix=lib/jquery-bootgrid https://github.com/rstaib/jquery-bootgrid.git master
       ```
-    - Add the code to integrate it in a separate commit.  Include:
-        - code to update it in Makefile
-	- Scrutinizer exclusions to .scrutinizer.yml (not needed if added to lib/ folder).
+    - Add the code to integrate it in a separate commit.
```

Comment:
I don't personally do this, it makes it a pain to test as you have to merge to branches to try things. I'd get rid.